contents/blog/introducing-hogql.md

```diff
@@ -0,0 +1,131 @@
+  - joe-martin
+tags:
+  - Product updates
+  - Release notes
+featuredImage: ../images/blog/hog_ql.png
+featuredImageType: full
+---
+
+Today, we're excited to announce a major new feature that's now available for all users as a public beta on PostHog Cloud - the ability to directly query data stored in PostHog via SQL.
+
+We've added several new ways to use SQL in PostHog - you can use expressions to enhance insights, you can use SQL to filter a revamped events list, and you can even use full SQL queries to analyze data in any way you want. 
+
+We've even created [our own SQL dialect, called HogQL](/docs/product-analytics/hogql).
+
+While PostHog's existing insights remain valuable for the vast majority of queries, HogQL gives them a massive upgrade for those times when off-the-shelf insights don't fit the bill. 
+
+Speaking of bills... these features are free for all users while in public beta – all we ask is that you [share any feedback with us](http://app.posthog.com/home#supportModal). 
+
+## What is HogQL?
+
+Basically, HogQL is our take on SQL. 
+
+Less basically, it's a transition layer over ClickHouse SQL that's intuitive for anyone familiar with SQL, but offers a few extra advantages and features.
+
+Unique features to HogQL include simplified access to event and person properties, as well as automatically added joins when you query fields with data on a different table, such as `events.person.properties.$browser`.
+
+You can also use subqueries, joins, table expressions, arrays, aggregations, lambdas and a whole host of other neat SQL features, including aggregations.
+
+Whether you're a technical product manager who needs multi-property breakdowns, or an engineer wishing to dissect data in more nuanced ways, HogQL unlocks a whole new world of insights to PostHog users.
+
+We recommend checking the docs for a more detailed [explanation of ClickHouse SQL functions which are supported in HogQL](/manual/hogql).
+
+## HogQL breakdowns
+
+![HogQL breakdowns](../images/blog/array/hog_breakdown.mp4)
+
+<Caption>Using a `properties.$browser_version` HogQL expression to breakdown results</Caption>
+
+A frequent request from teams using PostHog has been the ability to breakdown insight results across multiple properties. To accommodate this, we've added the ability to use HogQL expressions within a breakdown. 
+
+In other words, you can now break down results however you like – including by multiple properties at once. Want to break down new signups by both pricing tier _and_ overall usage? Now you can.
+
+> **Ready to get started?** Read our tutorial on [how to use HogQL for running advanced breakdowns](docs/tutorials/hogql-breakdowns)
+
+## HogQL filters
+
+![HogQL filter](../images/blog/array/hog_filters.mp4)
+
+<Caption>Using a `properties.$screen_width < properties.$screen_height` HogQL expression as a filter</Caption>
+
+Oh, look. You can use HogQL in filters too. Useful for _filtering_ by multiple properties!
+
+## HogQL aggregations
```

Comment:
I'm working on [this tutorial](https://github.com/PostHog/posthog.com/issues/6075) (almost done, but not a blocker), I've called it HogQL series. Do we think that is a better name? I'm neutral, can switch mine to aggregations.

Reply:
I have no opinion... thoughts @mariusandra?